src/applications/ivc-champva/10-7959C/config/form.js

```diff
@@ -29,38 +29,15 @@ import {
   applicantHasMedicareABSchema,
   applicantMedicareABContextSchema,
   applicantMedicarePartACarrierSchema,
-  applicantMedicarePartAEffectiveDateSchema,
   applicantMedicarePartBCarrierSchema,
-  applicantMedicarePartBEffectiveDateSchema,
   applicantMedicarePharmacySchema,
   applicantMedicareAdvantageSchema,
   applicantHasMedicareDSchema,
   applicantMedicarePartDCarrierSchema,
-  applicantMedicarePartDEffectiveDateSchema,
```

Comment:
We now collect these dates on the same page as the Part A/B/D provider name, so having separate schemas is no longer necessary.